### ISSUE TITLE
shadow-cljs fails to resolve clojure.spec.gen.alpha when building .cljc files

### ISSUE THREAD

**Issue by Jannis** [thheller/shadow-cljs#74]
Running shadow-cljs against a codebase with `.cljc` files that require `clojure` -> `cljs` aliased namespaces, shadow-cljs is unable to resolve these namespaces properly. The example I found the issue with was `clojure.spec.gen.alpha`:

```
~/W/w/entitydb (master|…) $ ./node_modules/.bin/shadow-cljs compile library
shadow-cljs - config: /Users/jannis/Work/workflo/entitydb/shadow-cljs.edn version: 1.0.20170721
shadow-cljs - starting ...
[:library] Compiling ...
-> Finding resources in classpath
<- Finding resources in classpath (258 ms)
ExecutionException: clojure.lang.ExceptionInfo: The required "clojure.spec.gen.alpha" is not available, required by "workflo/entitydb/core.cljc" {:tag :shadow.cljs.build/missing-ns, :ns clojure.spec.gen.alpha, :src "workflo/entitydb/core.cljc"}
	java.util.concurrent.FutureTask.report (FutureTask.java:122)
	java.util.concurrent.FutureTask.get (FutureTask.java:192)
	clojure.core/deref-future (core.clj:2297)
	clojure.core/future-call/reify--7899 (core.clj:6849)
	clojure.core/deref (core.clj:2317)
	clojure.core/deref (core.clj:2303)
	clojure.core/map/fn--5406 (core.clj:2733)
	clojure.lang.LazySeq.sval (LazySeq.java:40)
	clojure.lang.LazySeq.seq (LazySeq.java:49)
	clojure.lang.RT.seq (RT.java:525)
	clojure.core/seq--4944 (core.clj:137)
	clojure.core.protocols/seq-reduce (protocols.clj:24)
Caused by:
The required "clojure.spec.gen.alpha" is not available, required by "workflo/entitydb/core.cljc"
```

**Comment by thheller** [comment]
`shadow-cljs` uses a rather naive alias method in that it only aliases pre-defined namespaces. `cljs.spec.gen.alpha` just wasn't on the list of aliases.

Fixed in `shadow-cljs@0.18.5` and `[thheller/shadow-cljs "1.0.20170728"]`.

**Comment by thheller** [comment]
Forgot the reference in the commit msg: https://github.com/thheller/shadow-cljs/commit/651ad5935ee19a1e4099419311e59686cc2502ee
